rules/typescript/index.bzl

```diff
@@ -10,10 +10,11 @@ def _swc(**kwargs):
     )
 
 def ts_library(name, srcs, **kwargs):
+    # TODO: Enable isolated_typecheck = True for faster builds.
```

Comment:
@bduffany This would require additional type declarations in code, but may very well be worth it for the performance benefits. Worker mode is no longer supported with Typescript 5.